setup.py

```diff
@@ -42,9 +42,9 @@
     keywords='optics optical lens ray tracing matrix matrices aperture field stop\
     monte carlo design raytracing',
     packages=setuptools.find_packages(),
-    install_requires=['matplotlib'],
-    python_requires='>=3.6',
```

Comment:
Are we suppose to say 3.6 here or 3? SHouldn't that be 3.6 ? where else would we say it?

Reply:
This has to be 3.6 , a recent master merge changed it.